Doc/library/importlib.rst

```diff
@@ -307,7 +307,7 @@ ABC hierarchy::
 
     Loaders that wish to support resource reading should implement a
     :meth:`get_resource_reader` method as specified by
-    :class:`importlib.resources.abc.ResourceReader`.
+    :class:`importlib.resources.abc.TraversableResources`.
```

Comment:
No, it is documented in the `importlib.resources.abc.ResourceReader` description.

Reply:
But ResourceReader is scheduled for removal in 3.14

Reply:
When ResourceReader has been removed, the documentation of its methods and of `get_resource_reader()` should be moved to the TraversableResources description. For now, `get_resource_reader()` is described with ResourceReader, and the code that implements `get_resource_reader()` returning ResourceReader still works (but emits a warning).

Reply:
Not anymore, see the later comments in https://github.com/python/cpython/issues/97850